docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -157,6 +157,39 @@ children:
       - title: 'Database Migrations'
         url: Database-migrations.html
         output: 'web, pdf'
+        
+      - title: 'Working with data'
+        url: Working-with-data.html
+        output: 'web, pdf'
+        children:
+
+        - title: 'Querying data'
+        url: Querying-data.html
+        output: 'web, pdf'
```

Comment:
Do you intend to have `Querying data` as child of `Working with data? If so, you are missing `children:`.